treeherder/services/pulse/consumers.py

```diff
@@ -187,8 +189,39 @@ def on_message(self, body, message):
         message.ack()
 
 
+class JointConsumer(PulseConsumer):
+    queue_suffix = env("PULSE_QUEUE_NAME", default="queue")
+
+    def bindings(self):
+        rv = []
+        if self.source.get('hgmo'):
+            rv += HGMO_PUSH_BINDINGS
+        if self.source.get('github'):
+            rv += GITHUB_PUSH_BINDINGS
+        if self.source.get('pulse_url'):
+            rv += TASKCLUSTER_TASK_BINDINGS
+        return rv
+
+    @newrelic.agent.background_task(name='pulse-joint-listener.on_message', group='Pulse Listener')
+    def on_message(self, body, message):
+        exchange = message.delivery_info['exchange']
+        routing_key = message.delivery_info['routing_key']
+        logger.debug('received job message from %s#%s', exchange, routing_key)
+        if exchange == 'exchange/taskcluster-queue/v1/.':
+            store_pulse_tasks.apply_async(
+                args=[body, exchange, routing_key, self.root_url],
+                queue='store_pulse_tasks'
+            )
+        else:
+            store_pulse_pushes.apply_async(
+                args=[body, exchange, routing_key, self.root_url],
+                queue='store_pulse_pushes'
+            )
+        message.ack()
+
+
 class Consumers:
-    """
+    """pulse_listener_tasks_pushes: newrelic-admin run-program ./manage.py pulse_listener
```

Comment:
I would remove this comment as it is a Base class and the comment refers to a specific subclass.